style.css

```diff
@@ -93,10 +79,36 @@ img:hover{
   float: left;
 }
 
-#closing{
-  color:rgb(231,84,128);
+#closing {
+  color: rgb(231, 84, 128);
 }
 
-#candidates{
+#candidates {
   border: 10px solid pink;
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 600px) {
+  #tribute-link {
+    position: initial;
+    left: initial;
+    top: initial;
+  }
+  .column {
+    width: 100%;
+    float: none;

```

Comment:
If you're using a CSS Grid or Flexbox-based layout, you really don't need to use float. That property has been obsoleted by the more modern alternatives.